src/System.Reflection.MetadataLoadContext/src/System.Reflection.MetadataLoadContext.csproj

```diff
@@ -0,0 +1,160 @@
+    <Compile Include="System\Reflection\TypeLoading\Parameters\RoFatMethodParameter.cs" />
+    <Compile Include="System\Reflection\TypeLoading\Parameters\RoMethodParameter.cs" />
+    <Compile Include="System\Reflection\TypeLoading\Parameters\RoParameter.cs" />
+    <Compile Include="System\Reflection\TypeLoading\Parameters\RoPropertyIndexParameter.cs" />
+    <Compile Include="System\Reflection\TypeLoading\Parameters\RoThinMethodParameter.cs" />
+    <Compile Include="System\Reflection\TypeLoading\Properties\Ecma\EcmaProperty.cs" />
+    <Compile Include="System\Reflection\TypeLoading\Properties\RoProperty.cs" />
+    <Compile Include="System\Reflection\TypeLoading\RuntimeTypeInfo.BindingFlags.cs" />
+    <Compile Include="System\Reflection\TypeLoading\RuntimeTypeInfo.GetMember.cs" />
+    <Compile Include="System\Reflection\TypeLoading\RuntimeTypeInfo.TypeComponentsCache.cs" />
+    <Compile Include="System\Reflection\TypeLoading\Types\Ecma\EcmaDefinitionType.BindingFlags.cs" />
+    <Compile Include="System\Reflection\TypeLoading\Types\Ecma\EcmaDefinitionType.cs" />
+    <Compile Include="System\Reflection\TypeLoading\Types\Ecma\EcmaGenericMethodParameterType.cs" />
+    <Compile Include="System\Reflection\TypeLoading\Types\Ecma\EcmaGenericParameterType.cs" />
+    <Compile Include="System\Reflection\TypeLoading\Types\Ecma\EcmaGenericTypeParameterType.cs" />
+    <Compile Include="System\Reflection\TypeLoading\Types\RoArrayType.cs" />
+    <Compile Include="System\Reflection\TypeLoading\Types\RoArrayType.Key.cs" />
+    <Compile Include="System\Reflection\TypeLoading\Types\RoByRefType.cs" />
+    <Compile Include="System\Reflection\TypeLoading\Types\RoConstructedGenericType.cs" />
+    <Compile Include="System\Reflection\TypeLoading\Types\RoConstructedGenericType.Key.cs" />
+    <Compile Include="System\Reflection\TypeLoading\Types\RoDefinitionType.cs" />
+    <Compile Include="System\Reflection\TypeLoading\Types\RoExceptionType.cs" />
+    <Compile Include="System\Reflection\TypeLoading\Types\RoGenericParameterType.cs" />
+    <Compile Include="System\Reflection\TypeLoading\Types\RoHasElementType.cs" />
+    <Compile Include="System\Reflection\TypeLoading\Types\RoInstantiationProviderType.cs" />
+    <Compile Include="System\Reflection\TypeLoading\Types\RoModifiedType.cs" />
+    <Compile Include="System\Reflection\TypeLoading\Types\RoPinnedType.cs" />
+    <Compile Include="System\Reflection\TypeLoading\Types\RoPointerType.cs" />
+    <Compile Include="System\Reflection\TypeLoading\Types\RoStubType.cs" />
+    <Compile Include="System\Reflection\TypeLoading\Types\RoType.cs" />
+    <Compile Include="System\Reflection\TypeLoading\Types\RoType.GetInterface.cs" />
+    <Compile Include="System\Reflection\TypeLoading\Types\RoType.TypeClassification.cs" />
+    <Compile Include="System\Reflection\TypeLoading\Types\RoWrappedType.cs" />
+  </ItemGroup>
+  <ItemGroup>
```

Comment:
Add a condition for netcoreapp only:

`<ItemGroup Condition="'$(TargetGroup)' == 'netcoreapp'"`

Reply:
We have property for this: `<ItemGroup Condition="$(TargetsNetCoreApp)"`

Reply:
@joperezr could this cause compilation errors for netstandard if some of the assemblies referenced are not part of netstandard? The latest build did get some [errors ](https://ci3.dot.net/job/dotnet_corefx/job/master/job/windows-TGroup_netfx+CGroup_Release+AGroup_x86+TestOuter_false_prtest/18016/console)but at this point in time not sure what caused those. What does this condition do except remove unnecessary references to assemblies in the netstandard case?